setup-elyra.sh

```diff
@@ -14,7 +14,7 @@ export COS_BUCKET=${COS_BUCKET:-default}
 # If Kubeflow credentials are not supplied, use default Kubeflow installation credentials
 KF_USERNAME="${KF_USERNAME:=user@example.com}"
 KF_PASSWORD="${KF_PASSWORD:=12341234}"
-KF_DEPLOYMENT_NAMESPACE=$(env | grep JUPYTER_IMAGE | cut -d'/' -f2)
+KF_DEPLOYMENT_NAMESPACE=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
```

Comment:
This command does not seem to work in our OpenShift-based environment. I'm running the command on a working v0.1.3 image (our current fallback; we went straight from 0.1.3 to 0.2.1 before running into the issue here).

Reply:
hrmm, 
my current test env consists of a openshift cluster 4.10 with the 1.2.0 operator and 1.4.0 manifests.
Could you let me know what env you're running on so I can narrow down the issue?

Reply:
include any configuration changes like customized manifests too, thanks!

Reply:
Is it possible that in your environment the service account is limiting which secrets are exposed to the pod? The info in https://docs.openshift.com/container-platform/4.11/authentication/using-service-accounts-in-applications.html section "About automatically-generated service account token secrets" might also provide a clue.  Perhaps the output of `oc describe sa <service-account>` too.